docs/CNVs/CNV-methods.tex

```diff
@@ -1010,6 +1029,50 @@ \subsubsection{Results}
 \end{figure}
 
 
+\subsection{Inferring sex genotype from raw read counts}
+The probabilistic target coverage model requires the germline copy number of the targets as an input for dealing with cohorts composed of samples with different sexes. If the phenotypic sex of the human subject associated to a sample is known, the nominal germline ploidy of each target is readily known. Unfortunately, the gender annotation is not available for most samples. In this section, we propose a simple statistical test for inferring the sex associated to a sample from human subjects from raw read counts on X and Y chromosomes.\\
+
+Let $T_X$ and $T_Y$ be a set of targets on X and Y chromosomes. Let $\{n_t\}$ be the raw read counts on sex chromosome targets. The likelihood of reads for either of XX or XY genotypes can be calculated from Eq.~\eqref{coverage_model}. Since we have no prior knowledge of the germline CNV events\footnote{The germline CNV events will be known only after training the target coverage model. The latter, however, requires the sex genotype as an input.}, we proceed by assuming $c_{st} = 1$. We have no prior knowledge about the multiplicative bias $e^{b_t}$ either. However, it is reasonable to assume that a major source of the mean target bias is the target length $l_t$. In particular, if the reads are corrected for the GC bias, we expect a strong $e^{b_t} \propto l_t$ proportionality. Finally, the average read depth, $d$, can be reliably estimated from the read counts on autosomal targets (see Sec.~\ref{sec:read_depth_estimation}). Put together, for the XX genotype we have:
+\begin{equation}
+\text{(XX~genotype)} \qquad \qquad n_t \sim \left\{
+\begin{array}{ll}
+	\mathrm{Poisson}(2\,\rho\,l_t) & \qquad t \in T_X,\\
+	\\
+	\mathrm{Poisson}(\epss_M\,l_t) & \qquad t \in T_Y,\\	
+\end{array}
+\right.
+\end{equation}
+where $\rho$ is {\em average read count per base per strand}, $l_t$ is the target length (number of overlapping bases), and $\epss_M \sim 10^{-5}$ is the (small) probability of a mapping error that may result in a (small) number of reads to be mapped to the Y chromosome for an XX genotype sample. Similarly, for the YY genotype we have:
```

Comment:
What I mentioned (1e-5) was a typical number -- $\epss_M$ comes from mapping quality. Once we get from BAM files to ReadCountCollection, mapping quality information is lost. In practice, a ballpark estimate is sufficient though: XX and XY samples are sufficiently easy to tell apart. That being said, if we set $\epss_M = 0$, we always get XY genotype since all XX samples have a small number of their reads erroneously aligned to Y chr. I am not sure if I understand the role of pseudo-autosomal regions -- do you mean somehow we expect a higher mapping error rate, compared to autosomal targets, in the pseudo-autosomal regions of X and Y chromosomes?